glotaran/testing/parallel_spectral_decay.py

```diff
@@ -0,0 +1,68 @@
+"""A simple parallel decay for testing purposes."""
+import numpy as np
+
+from glotaran.analysis.simulation import simulate
+from glotaran.io import load_model
+from glotaran.io import load_parameters
+from glotaran.project import Scheme
+from glotaran.project.generators import generate_model_yml
+
+SIMULATION_MODEL_YML = generate_model_yml(
+    "spectral-decay-parallel", **{"nr_compartments": 3, "irf": True}
+)
+SIMULATION_MODEL = load_model(SIMULATION_MODEL_YML, format_name="yml_str")
+
+MODEL_YML = generate_model_yml("decay-parallel", **{"nr_compartments": 3, "irf": True})
+MODEL = load_model(MODEL_YML, format_name="yml_str")
+
+WANTED_PARAMETER_YML = """
```

Comment:
`SIMULATION_PARAMETER_YML` would be a more fitting name since the actually wanted parameters are `PARAMETER_YML`.
That said we could also reuse the `PARAMETER_YML` To reduce duplication and improve readability.